src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,18 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
+    private boolean isPrivate;
+
+	public Contact(boolean isPrivate) {
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically. See here [http://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs](url)